config/theme-tabler.php

```diff
@@ -19,22 +19,10 @@
      * Simple pick a layout and let Backpack decide the best look for it.
      * No extra step is required.
      *
-     * Where should the menu be positioned?
+     * Possible values: horizontal, horizontal_dark, horizontal_overlap, vertical,
+     * vertical_dark, vertical_transparent, right_vertical, right_vertical_dark, right_vertical_transparent
      */
     'layout' => 'horizontal',
-    // horizontal
-    // horizontal_dark
-    // horizontal_overlap
-    // vertical
-    // vertical_dark
-    // vertical_transparent
```

Comment:
The `vertical`, `vertical_dark` and `vertical_transparent` layouts we have include a topbar. While CoreUI doesn't, in those same layouts:
![CleanShot 2023-04-19 at 10 57 19@2x](https://user-images.githubusercontent.com/1032474/233008281-14f39596-1d8a-4a13-a4e6-1238a79d6cd0.png)
![CleanShot 2023-04-19 at 10 58 04@2x](https://user-images.githubusercontent.com/1032474/233008289-941fc06c-bf1c-405e-a45a-6c74bde054f2.png)

**Why is that there - I wondered?** Well historically, that top bar only had one purpose in Backpack: having the user dropdown in the top-right corner (which is intuitive). Now in Tabler it has an extra purpose... the light/dark toggle. So it _kind of_ makes sense for us to keep a topbar.

BUT.

(1) **It definitely, 100%, should not be fixed to the top. Ever. At least not by default.** 😀 It is very rare that I find something important enough to be fixed. If you ask me, 99% of elements/menus/etc are not important enough. Why? Fixing something to the screen means it takes up that visual real estate... for ever, for all pages! So if it takes up 10% of the page... we are making a decision that that element is so _freaking_ important, and the user will use it _so freaking often_ that it makes sense for it to be on the screen all the time, and take up 10% of the screen all the time. Sometimes that is true. But in 99% of cases... it's not. In our case... is the Logout and navbar toggle so important... so often used... that it takes up 10% of all screens? I don't think so. In fact, I think the Logout button will almost never be used... and the Light/Dark toggle will only be used once per user (not even per session). That's why I'm pretty sure... we should NOT have the top navbar fixed, in any of our views.

(2) **Should we even have a topbar in these layouts?** If those particular elements (Logout & light/dark toggle) are so unimportant... why place them so high up the visual hierarchy? Couldn't we include them in the sidebar? Perhaps... separately, at the end of the sidebar? Which means in the bottom-left corner? That would completely eliminate the need for the topbar, and create a super-clear visual separation: left is for menus, right is for content. Just like the Tabler layouts have.